lib/cli/parser-helpers.js

```diff
@@ -28,6 +28,17 @@ function parseSecurityFeatures (features) {
   return parsedFeatures;
 }
 
+function parsePluginNames (names) {
+  const splitter = (splitOn, str) => `${str}`.split(splitOn).map((s) => s.trim()).filter(Boolean);
```

Comment:
I don't see a need in this method. Also, an exception here might only be thrown if `names` is not a string or it is empty/blank. I would prefer an explicit check though

Reply:
empty/blank string would be safe here. but i'll add a check for non-string.